test/libsolidity/semanticTests/end-to-end/balance.sol

```diff
@@ -0,0 +1,13 @@
+contract test {
+    constructor() public payable {}
+
+    function getBalance() public returns (uint256 balance) {
+        return address(this).balance;
+    }
+}
+
+// ====
+// compileViaYul: also
+// ----
+// constructor(), 23 ether ->
+// getBalance() -> 23000000000000000000
```

Comment:
Is that the expectation that was computed by the extractor? It used to be just "23" in the input file.

Reply:
No this was just a manual change with the help of `isoltest`. Will change to `wei`.